### PR TITLE
fix(teamStats): Sentence case empty state text on team stats

### PR DESCRIPTION
**Pull request by scttcper**

### CODE REVIEW

File: static/app/views/organizationStats/teamInsights/teamAlertsTriggered.tsx [hunk 1]
```diff
@@ -152,7 +152,7 @@ class TeamAlertsTriggered extends AsyncComponent<Props, State> {
           isEmpty={
             !alertsTriggered || !alertsTriggeredRules || alertsTriggeredRules.length === 0
           }
-          emptyMessage={t("No Alerts Triggered For This Team's Projects")}
+          emptyMessage={t("No alerts triggered for this team's projects")}
```
Comment (robinrendle) [inline static/app/views/organizationStats/teamInsights/teamAlertsTriggered.tsx:155]: Can we change this to `’`? 😬

File: static/app/views/organizationStats/teamInsights/teamReleases.tsx [hunk 2]
```diff
@@ -245,7 +245,7 @@ class TeamReleases extends AsyncComponent<Props, State> {
         </ChartWrapper>
         <StyledPanelTable
           isEmpty={projects.length === 0}
-          emptyMessage={t("No Releases Were Setup For This Team's Projects")}
+          emptyMessage={t("No releases were setup for this team's projects")}
```
Comment (robinrendle) [inline static/app/views/organizationStats/teamInsights/teamReleases.tsx:248]: Same here

File: static/app/views/organizationStats/teamInsights/teamIssuesAge.tsx [hunk 3]
```diff
@@ -137,7 +137,7 @@ class TeamIssuesAge extends AsyncComponent<Props, State> {
         </ChartWrapper>
         <StyledPanelTable
           isEmpty={!oldestIssues || oldestIssues.length === 0}
-          emptyMessage={t("No Unresolved Issues For This Team's Projects")}
+          emptyMessage={t("No unresolved issues for this team's projects")}
```
Comment (robinrendle) [inline static/app/views/organizationStats/teamInsights/teamIssuesAge.tsx:140]: Same here

File: static/app/views/organizationStats/teamInsights/teamAlertsTriggered.tsx (hunk 1 above)
Comment (evanpurkhiser) [reply static/app/views/organizationStats/teamInsights/teamAlertsTriggered.tsx]: good luck fixing this everywhere @robinrendle 😱

**Review by robinrendle — APPROVED** [review]

**Pull request merged**